web/src/components/tasks/business-structure/BusinessStructureTask.tsx

```diff
@@ -155,7 +156,9 @@ export const BusinessStructureTask = (props: Props): ReactElement => {
       )}
       {business && !showRadioQuestion && (
         <>
-          <h3>{Config.businessStructureTask.completedHeader}</h3>
+          <Heading level={2} styleVariant="h3">
```

Comment:
Good catch! This is actually part of the story rather than the refactor, but I left it in since it more directly relates to the refactor than the other story items.